src/m2n/tests/PointToPointCommunicationTest.cpp

```diff
@@ -419,6 +419,276 @@ BOOST_AUTO_TEST_CASE(ConnectionTest, * testing::OnSize(4))
   
 }
 
+BOOST_AUTO_TEST_CASE(P2PMeshBroadcastTest, * testing::OnSize(4))
+{
+
+  assertion(utils::Parallel::getCommunicatorSize() == 4);
+
+  // com::PtrCommunicationFactory cf(new com::SocketCommunicationFactory);
+  com::PtrCommunicationFactory cf(new com::MPIPortsCommunicationFactory); 
```

Comment:
If you use MPI Ports you need to define the MPI Ports label `* boost::unit_test::label("MPI_Ports")`. As we exclude those tests for many systems though, it's probably better to use the sockets communication.